test/input-validation.js

```diff
@@ -139,7 +139,7 @@ module.exports.payloadMethod = function (method, t) {
       t.error(err)
     }
```

Comment:
this `if` around `t.error` can likely be removed following what's done in most other test files.